src/components/ColumnLeft/CredentialSetItem/CredentialSetItem.js

```diff
@@ -15,11 +21,52 @@ export default function CredentialSetItem({
       type,
       title,
       subtitle,
-      expiration_dateint: d,
+      expiration_dateint: expiry,
     },
   },
   index,
+  records,
 }) {
+  const [isDisconnected, setIsDisconnected] = useState(false)
+  const [lastProposalId, setLastProposalId] = useState('')
+  const [statusConnId, errorConnId, startFetchConnIdHandler] = useGetConn()
+  const [errorProposal, startFetchHandler] = usePostPresentProofSendProposal()
+  const [statusSendProposal, setStatusSendProposal] = useState('')
```

Comment:
just a suggestion, and I believe this is something per taste bassis. But we could have one hook containing, all 3 props
- isDisconnected
- lastProposolId
- status

```js
const [ hook, updateHook] = React.usState({
  status: 'dead',
  isDisconnected: true,
});

...

updateHook({ ...hook, newState })  // in some cases
updatedHook((prevState) => ({ ...hook, opt1: !prevState }))
```

I just found easier when they are combined, but for some it might be easier when they are individual

Reply:
moved to a new component but kept as separate useStates